extensions/microsoft-authentication/src/AADHelper.ts

```diff
@@ -20,7 +20,7 @@ import { MicrosoftAuthenticationSession } from './microsoft-authentication';
 
 const localize = nls.loadMessageBundle();
 
-const redirectUrl = 'https://vscode-redirect.azurewebsites.net/';
+const redirectUrl = 'http://localhost/redirect';
```

Comment:
Is this even used anywhere? Since we use our own custom account handler I didn't think there was anywhere in the app users could actually sign in to their Microsoft account.

Reply:
I don't think this is used anywhere.

Reply:
This should have a SQL CARBON EDIT. But if it's not used anywhere I would suggest just reverting it - the fewer changes we make to VS Code stuff the easier merges are.